index.js

```diff
@@ -35,7 +35,17 @@ const pkg = getPackageJson();
   const tagSuffix = process.env['INPUT_TAG-SUFFIX'] || '';
   console.log('tagPrefix:', tagPrefix);
   console.log('tagSuffix:', tagSuffix);
-  const messages = event.commits ? event.commits.map((commit) => commit.message + '\n' + commit.body) : [];
+
+  const checkLastCommitOnly = process.env['INPUT_CHECK-LAST-COMMIT-ONLY'] || 'false';
+
+  let messages = []
+  if (checkLastCommitOnly === 'true') {
+    console.log('Only checking the last commit...');
+    const commit = event.commits ? event.commits[event.commits.length - 1] : null;
```

Comment:
should we do a check for this, if there is `commits.lengths > 0`?

Reply:
Thank you for the comment @phips28! Sure, would it be ok to check for `event.commits && events.commits.lengths > 0` in your opinion?

Reply:
yes

Reply:
I updated the code accordingly, Thank you!